assets/scripts/segments/components.json

```diff
@@ -30,6 +30,33 @@
     "sidewalk": {
       "elevation": 1,
       "variants": {
+        "color": {
+          "regular": {
+            "graphics": {
+              "repeat": "ground--concrete"
+            }
+          },
+          "black": {
+            "graphics": {
+              "repeat": "ground--asphalt"
+            }
+          },
+          "red": {
+            "graphics": {
+              "repeat": "ground--asphalt-red"
+            }
+          },
+          "green": {
+            "graphics": {
+              "repeat": "ground--asphalt-green"
+            }
+          },
+          "gray": {
+            "graphics": {
+              "repeat": "ground--asphalt-gray"
+            }
```

Comment:
We may have an issue where the colored ground assets will be off because they weren't initially designed to be at sidewalk level. Do you have a screenshot of this working?

Reply:
<img width="411" alt="Screen Shot 2020-03-04 at 22 23 59" src="https://user-images.githubusercontent.com/16242142/75944968-fcd95c80-5e66-11ea-9992-0220b455670e.png">
<img width="436" alt="Screen Shot 2020-03-04 at 22 24 05" src="https://user-images.githubusercontent.com/16242142/75944969-fd71f300-5e66-11ea-9af5-d8b9ee384f78.png">

It seems to work for all of the three colors that work with bike lanes

Reply:
That's awesome - thanks!